forte/api/forte_python_module.cc

```diff
@@ -231,6 +231,7 @@ PYBIND11_MODULE(forte, m) {
                       std::shared_ptr<psi::Vector>, std::shared_ptr<psi::Vector>>())
         .def("nmopi", &SCFInfo::nmopi, "the number of orbitals per irrep")
         .def("doccpi", &SCFInfo::doccpi, "the number of doubly occupied orbitals per irrep")
+        .def("set_doccpi", &SCFInfo::set_doccpi, "set the number of doubly occupied orbitals per irrep")
```

Comment:
Please remove this function. As you can see this class does not have any setter functions because it is intended to be set and passed without allowing modifications. Instead of using `set_doccpi`, create a new SCFInfo object with a different doccpi value.